sdk/template/azure-template-stress/Chart.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v2
+name: java-template
+description: An example stress test chart for performing azure resource deployments
+version: 0.1.1
+appVersion: v0.1
+annotations:
+  stressTest: 'false'  # change it to true. This enables auto-discovery of this test via `find-all-stress-packages.ps1`
```

Comment:
How come we are defaulting to false here? To prevent auto-discovery during development? It seems like it could be an odd behavior to debug if the script isn't finding one's test.

Reply:
this is a template/example project which exist for documentation purposes. While the test is still useful, we don't really want to run it in stress pipeline